mindjunkies/forums/models.py

```
@@ -34,76 +39,76 @@ def save(self, *args, **kwargs):
+        except:
+            return f'no author : {self.body[:30]}'
 
     class Meta:
-        ordering = ["-created_at"]
+        ordering = ['created']
 
+class LikedReply(models.Model):
+    reply = models.ForeignKey(Reply, on_delete=models.CASCADE)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    created = models.DateTimeField(auto_now_add=True)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)